src/source/kafka.rs

```diff
@@ -243,8 +243,14 @@ impl Source for Int {
                         m.offset().to_string(),
                     ];
                     let data = data.to_vec();
+                    let mut kafka_meta_data = Value::object_with_capacity(1);
+                    kafka_meta_data.insert("kafka", Value::object_with_capacity(2))?;
+                    if let Some(key) = m.key() {
+                        if let Some(meta_data) = kafka_meta_data.get_mut("kafka") {
+                            meta_data.insert("key", Vec::from(key))?;
```

Comment:
The key will be a `Value::Bytes` right? Could you create it as such here, just for clarity?
```suggestion
                            meta_data.insert("key", Value::Bytes(Vec::from(key).into())?;
```